websites/B/BreathFM/presence.ts

```diff
@@ -0,0 +1,59 @@
+const presence = new Presence({
+		clientId: "846282107462352927",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+	let presenceData: PresenceData;
+	if ("mediaSession" in navigator && navigator.mediaSession.metadata !== null) {
+		presenceData = {
+			largeImageKey: navigator.mediaSession.metadata.artwork[0].src,
+			details: navigator.mediaSession.metadata.title,
+			state: navigator.mediaSession.metadata.artist,
+		};
```

Comment:
Also optional but add buttons (Don't know how to format this, sorry)
```suggestion
			buttons: [
							{
								label: "Listen",
								url: location.href,
							},
						];
		};
```